language-extensions/dotnet-core-CSharp/sample/SubModuleCall/SubModule.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace UserExecutor
+{
+
+    //Lets assume that a DLL is created using this kind of code in another project. 
+    // The DLL is moved into the folder where access is granted. 
+    // Access needs to be granted to respective folder before you execute these commands. 
+    // To grant access perform below commands in windows command prompt. 
+    // Note: The folder name may change in your environment. So please specify the right folder name.
+    // Command1: 
+    //          icacls "C:\Program Files\Microsoft SQL Server\MSSQL16.SQLSERVER2022\MSSQL\ExternalLibraries\6\65537\1" /grant "SQLRUsergroupSQLSERVER2022":(OI)(CI)RX /T  
+    // Change the server name according to your naming convension .
+    // Command2:
+    //          icacls "C:\Program Files\Microsoft SQL Server\MSSQL16.SQLSERVER2022\MSSQL\ExternalLibraries\6\65537\1"  /grant *S-1-15-2-1:(OI)(CI)RX /T
+    // After executing above commands, SQL server will have access to local folder which has been specified in the commands. 
+    // All these commands are available in  OneTimeSetupCommands.cmd file in this solution as well
+    //
+    public class SubModule
```

Comment:
May I know what this class does?

Reply:
This is just an sample class.. This will be replaced by whichever DLL customer has. Just to show a complete story we have included a simple "Hello World" Kind of module.

Reply:
So instead of a real sample, this is more like a template? Have you tried to compile and run an E2E sample like the [regex sample](https://github.com/microsoft/sql-server-language-extensions/tree/main/language-extensions/dotnet-core-CSharp/sample/regex)? Personally I find real E2E samples more helpful if I am new to this. If you did, I would recommend providing the real sample along with the sql scripts.